tools/sdk-generation-pipeline/pipelines/steps/generate-sdk.yml

```diff
@@ -33,33 +36,42 @@ steps:
     condition: ne('${{ parameters.triggerType }}', 'release')
     
   - bash: |
-      if [ "${{ parameters.serviceType }}" = "resource-manager" ];then
+      if [ !  -z "${{ parameters.cadlProject }}" ]; then
```

Comment:
`  - name: relativeCadlProjectPath
    type: string
    default: "''"
`

${{ parameters.cadlProject }} = ''
`if [ ! -z "${{ parameters.cadlProject }}" ]`
may the judgement function always be  true